daml-lf/spec/daml-lf-1.rst

```diff
@@ -1822,6 +1851,17 @@ as described by the ledger model::
        ⇓ᵤ
      Err "Exercise actors do not match"  ‖ E₁ ; (st; keys)
 
+     'tpl' (x : T)
+         ↦ { 'choices' { …, 'choice' ChKind Ch (y : τ) (z) : σ  'by' eₚ ↦ eₐ, … }, … }  ∈  〚Ξ〛Mod
```

Comment:
what's the `(z)`?

Reply:
It's for the contract id of the contract the choice is exercised on, called `self` in the surface language. It's the same in the rule above.